gusto/timeloop.py

```diff
@@ -460,8 +462,8 @@ def __init__(self, equation_set, io, transport_schemes, spatial_methods,
 
         for _, scheme in self.all_physics_schemes:
             assert scheme.nlevels == 1, "multilevel schemes not supported as part of this timestepping loop"
-            assert isinstance(scheme, ExplicitTimeDiscretisation), \
-                "Only explicit time discretisations can be used for physics"
+            # assert isinstance(scheme, ExplicitTimeDiscretisation), \
```

Comment:
This check needs to become a check for whether the time discretisation is explicit and whether this is allowed for the physics scheme (add a logical to the physics scheme).